pom.xml

```diff
@@ -27,7 +27,7 @@
 		<url/>
 	</scm>
 	<properties>
-		<java.version>21</java.version>
+		<java.version>17</java.version>
```

Comment:
A few of us won't have java 17. Can you just download 21 so that we're all on the same version?

Reply:
This is currently a breaking change. We should try not to introduce breaking changes in future versions .